deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -769,3 +768,17 @@ async def get_versions(request: Request):
     except Exception as e:
         logger.error(f"Failed to retrieve info of pulse3d versions: {repr(e)}")
         raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@app.get("/usage-quota", response_model=Union[UsageQuota, GenericErrorResponse])
+async def get_usage_quota(request: Request, token=Depends(ProtectedAny(scope=PULSE3D_USER_SCOPES))):
+    """Get the usage quota for the spesific user"""
+    try:
+        customer_id = str(uuid.UUID(token["customer_id"]))
```

Comment:
and for customer accounts, this will be the `userid` instead of `customer_id`, so once these changes are made, doubel check this works for both account types